src/testrel.jl

```diff
@@ -26,11 +26,13 @@ function set_context(new_context::Context)
     return TEST_CONTEXT_WRAPPER.context = new_context
 end
 
-function create_test_database(clone_db::Union{Nothing, String} = nothing)::String
+function create_test_database_name()::String
     basename = get(ENV, "TEST_REL_DB_BASENAME", "test_rel")
-    schema = gen_safe_name(basename)
+    return gen_safe_name(basename)
+end
 
-    return create_database(get_context(), schema; source = clone_db).database.name
+function create_test_database(name::String, clone_db::Union{Nothing, String} = nothing)::String
+    return create_database(get_context(), name; source = clone_db).database.name
 end
```

Comment:
presumably, this returns the name we just passed in - is this the case? if so, do we still need to return it?

Reply:
It's outdated. Originally it was something like `create_db(gen_name())` It makes less sense now. I changed the code to assume the database created is the one asked for.